AndroidNativeKotlinTemplate/app/src/main/AndroidManifest.xml

```diff
@@ -48,21 +48,9 @@
         -->
         <!-- Use `android:name="com.salesforce.androidnativekotlintemplate.QrCodeEnabledLoginActivity"` when enabling log in via Salesforce UI Bridge API generated QR codes -->
 <!--        <activity-->
-<!--            android:name="com.salesforce.androidnativekotlintemplate.QrCodeEnabledLoginActivity"-->
-<!--            android:theme="@style/SalesforceSDK"-->
+<!--            android:name="com.salesforce.androidsdk.ui.LoginActivity"-->
```

Comment:
This should be LoginActivity since this block is for the browser based auth comment flow.